revolv/donor/views.py

```diff
@@ -18,6 +20,12 @@ def dispatch(self, request, *args, **kwargs):
 
         return super(DonorDashboardView, self).dispatch(request, *args, **kwargs)
 
+    def make_statistics_dictionary(self):
```

Comment:
This function is copied twice in both views, so it looks like you could make it take a profile as an argument and put it in a utility function e.g. `project.utils.aggregate_stats` or something like that. If you review #355 and we merge it before this then that function could go in revolv/project/stats.py :D